ansible/roles/test/tasks/add_container_to_inventory.yml

```diff
@@ -0,0 +1,24 @@
+- name: Add/update the public key in the "{{ lookup('env','HOME') + '/.ssh/known_hosts' }}"
+  delegate_to: localhost
+  known_hosts:
+    name: "{{ ansible_host }}"
+    key: "{{ lookup('pipe', 'ssh-keyscan ' + ansible_host) }}"
+    state: "present"
+
+- name: Generate an OpenSSH keypair
+  delegate_to: localhost
+  openssh_keypair:
+    path: "{{ lookup('env','HOME') + '/.ssh/id_rsa' }}"
+
+- name: Set authorized key
+  authorized_key:
+    user: "{{ ansible_ssh_user }}"
+    state: present
+    key: "{{ lookup('file', lookup('env','HOME') + '/.ssh/id_rsa.pub') }}"
+
+- name: Add {{ container_name }} container to inventory
+  add_host:
+    name: "{{ container_name }}"
```

Comment:
>container_name [](start = 14, length = 14)

Can we name it as {{ansible_host}}_{{container_name}}?

Reply:
Sorry I could not get your point. My intention is to give a more specific name to prevent conflicting names in inventory.

---
In reply to: [356375708](https://github.com/Azure/sonic-mgmt/pull/1269#discussion_r356375708) [](ancestors = 356375708)

Reply:
Done